sportnet/sportnet/playgrounds/models.py

```diff
@@ -4,14 +4,24 @@
 
 
 class PlayGround(TimeStampedModel):
+
+    title_action = models.CharField(max_length=128, verbose_name='Название мероприятия', blank=False, null=False)
```

Comment:
достаточно просто `title`
надо подумать, нужна ли тут уникальность еще? А то мало ли, кто то наплодит мероприятий с одинаковым названием, пользователь будет путаться.
```suggestion
    title = models.CharField(max_length=128, verbose_name='Название', blank=False, null=False, unique=True)
```